grammar.js

```diff
@@ -807,16 +833,13 @@ module.exports = grammar({
     )),
 
     enumerators: $ => seq(
-      sep1($._semicolon, choice(
-        $.enumerator,
-        $.assignment_expression
-      )),
+      sep1($._semicolon, $.enumerator),
       optional($._automatic_semicolon)
     ),
 
     enumerator: $ => seq(
-      $.identifier,
-      '<-',
+      $._pattern,
+      choice('<-', '='),
```

Comment:
apologies for the dumb question. do you mind spelling out for me a scala example of what this change achieves